.ci_support/osx_arm64_python3.9.____cpython.yaml

```diff
@@ -5,7 +5,7 @@ c_compiler:
 c_compiler_version:
 - '11'
 channel_sources:
-- conda-forge
+- conda-forge/label/rust_dev,conda-forge
```

Comment:
Including `rust_dev` here seems...odd?

Reply:
We add that for all `osx-arm64` builds as currently only the rust nightlies are working there.

Reply:
That's done by the re-render ( https://github.com/conda-forge/python.app-feedstock/pull/21/commits/4f7bb8d92b7f9a28fee1196a635de271f21d8c08 ) and [comes from `conda-forge-pinning`]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/7af3a80bf666ea2b6af95096404b237f63554b52/recipe/conda_build_config.yaml#L105 ). I think this has to do with some issues building Rust on macOS ARM, but haven't followed that work closely

Reply:
Ah, now it makes sense. Thanks for the clarification.